pallets/funding/src/tests/1_application.rs

```diff
@@ -0,0 +1,1321 @@
+				assert_ok!(inst.execute(|| {
+					Pallet::<TestRuntime>::create_project(RuntimeOrigin::signed(issuer), jwt, project_metadata.clone())
+				}));
+			}
+
+			#[test]
+			fn multiple_creations_different_issuers() {
+				let mut inst = MockInstantiator::new(Some(RefCell::new(new_test_ext())));
+				let mut issuer = ISSUER_1;
+				for _ in 0..512 {
+					let project_metadata = default_project_metadata(inst.get_new_nonce(), issuer);
+					inst.create_evaluating_project(project_metadata, issuer);
+					inst.advance_time(1u64).unwrap();
+					issuer += 1;
+				}
+			}
+
+			#[test]
+			fn multiple_funding_currencies() {
+				let mut inst = MockInstantiator::new(Some(RefCell::new(new_test_ext())));
+				let mut counter: u8 = 1u8;
+				let mut with_different_metadata = |mut project: ProjectMetadataOf<TestRuntime>| {
+					let mut binding = project.offchain_information_hash.unwrap();
+					let h256_bytes = binding.as_fixed_bytes_mut();
+					h256_bytes[0] = counter;
+					counter += 1u8;
+					project.offchain_information_hash = Some(binding);
+					project
+				};
+				let default_project_metadata = default_project_metadata(inst.get_new_nonce(), ISSUER_1);
+
+				let mut one_currency_1 = default_project_metadata.clone();
+				one_currency_1.participation_currencies = vec![AcceptedFundingAsset::USDT].try_into().unwrap();
+
+				let mut one_currency_2 = default_project_metadata.clone();
+				one_currency_2.participation_currencies = vec![AcceptedFundingAsset::USDC].try_into().unwrap();
+
+				let mut one_currency_3 = default_project_metadata.clone();
+				one_currency_3.participation_currencies = vec![AcceptedFundingAsset::DOT].try_into().unwrap();
+
+				let mut two_currencies_1 = default_project_metadata.clone();
+				two_currencies_1.participation_currencies =
+					vec![AcceptedFundingAsset::USDT, AcceptedFundingAsset::USDC].try_into().unwrap();
+
+				let mut two_currencies_2 = default_project_metadata.clone();
+				two_currencies_2.participation_currencies =
+					vec![AcceptedFundingAsset::USDT, AcceptedFundingAsset::DOT].try_into().unwrap();
+
+				let mut two_currencies_3 = default_project_metadata.clone();
+				two_currencies_3.participation_currencies =
+					vec![AcceptedFundingAsset::USDC, AcceptedFundingAsset::DOT].try_into().unwrap();
+
+				let mut three_currencies = default_project_metadata.clone();
+				three_currencies.participation_currencies =
+					vec![AcceptedFundingAsset::USDT, AcceptedFundingAsset::USDC, AcceptedFundingAsset::DOT]
+						.try_into()
+						.unwrap();
+
+				let projects = vec![
+					one_currency_1.clone(),
+					one_currency_2.clone(),
+					one_currency_3,
+					two_currencies_1,
+					two_currencies_2,
+					two_currencies_3,
+					three_currencies,
+				];
+
+				let test_1 = with_different_metadata(one_currency_1);
+				let test_2 = with_different_metadata(one_currency_2);
+				assert!(test_1.offchain_information_hash != test_2.offchain_information_hash);
```

Comment:
This assert was there because `with_different_metadata` was not behaving as expected at first. But now that we removed the requirement of having unique hashes on each project we can remove all this